package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A lego block programming language",
   "main": "run_server.js",
   "scripts": {
-    "test": "./node_modules/.bin/cucumber.js",
+    "test": "xvfb-run ./node_modules/.bin/cucumber.js",
```

Comment:
Tests now require the `xvfb-run` command before running. (for frontend testing).
- It is installed in the cloud 9 workspace and has been aliased to just `cucumber`
- so you can run tests using just `cucumber`  or `cucumber /features/test/testdir/testname.feature`